src/platform/qt/SettingsView.cpp

```diff
@@ -596,12 +596,12 @@ void SettingsView::reloadConfig() {
 	loadSetting("cheatsPath", m_ui.cheatsPath);
 	loadSetting("showLibrary", m_ui.showLibrary);
 	loadSetting("preload", m_ui.preload);
-	loadSetting("showFps", m_ui.showFps, true);
-	loadSetting("cheatAutoload", m_ui.cheatAutoload, true);
-	loadSetting("cheatAutosave", m_ui.cheatAutosave, true);
```

Comment:
This should not be removed; it's for setting the default when no option is present and entirely unrelated to the bug in question. Why did you remove it?